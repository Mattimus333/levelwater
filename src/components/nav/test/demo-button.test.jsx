import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import DemoButton from '../demo-button.jsx';

describe('oauth button', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<DemoButton />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
