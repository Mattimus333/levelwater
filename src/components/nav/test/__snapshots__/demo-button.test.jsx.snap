exports[`oauth button renders a snapshot 1`] = `
<button
  className="ui button">
  Try It!
</button>
`;
